Assets/Scripts/ssjAug2018/DeilveryTarget/DeliveryTargetSpawnManager.cs

```diff
@@ -0,0 +1,36 @@
+﻿using pdxpartyparrot.Core.Util;
```

Comment:
No need for this class at all. DeliveryTargetManager should have all of the functionality